### PR TITLE
Update benchmark weights

### PR DESCRIPTION
**Pull request by HenriqueNogara**
Update benchmarks

### CODE REVIEW

File: pallets/weights/src/pallet_compliance_manager.rs [hunk 1]
```diff
@@ -57,7 +57,7 @@ impl pallet_compliance_manager::WeightInfo for SubstrateWeight {
     /// The range of component `b` is `[1, 10]`.
     /// The range of component `c` is `[1, 10]`.
     /// The range of component `d` is `[1, 10]`.
-    fn condition_costs(a: u32, _b: u32, c: u32, _d: u32, ) -> Weight {
+    fn condition_costs(a: u32, _b: u32, c: u32, _d: u32) -> Weight {
```
Comment (HenriqueNogara) [inline pallets/weights/src/pallet_compliance_manager.rs]: Ignored parameters

File: pallets/weights/src/pallet_identity.rs [hunk 2]
```diff
@@ -54,269 +54,364 @@ use polymesh_runtime_common::{RocksDbWeight as DbWeight, Weight};
...
-            .saturating_add(Weight::from_ref_time(100_000).saturating_mul(p.into()))
-            // Standard Error: 140_388
-            .saturating_add(Weight::from_ref_time(12_202_214).saturating_mul(l.into()))
-            // Standard Error: 140_388
-            .saturating_add(Weight::from_ref_time(11_817_991).saturating_mul(e.into()))
+    /// The range of component `a` is `[0, 2000]`.
+    /// The range of component `p` is `[0, 2000]`.
+    /// The range of component `l` is `[0, 80]`.
+    /// The range of component `e` is `[0, 80]`.
+    fn permissions_cost(_a: u32, _p: u32, l: u32, e: u32) -> Weight {
```
Comment (HenriqueNogara) [inline pallets/weights/src/pallet_identity.rs]: Ignored parameters, but has been addressed in: pallets/common/src/traits/identity.rs by adding this: 
```
if assets > MAX_ASSETS as u32
   || portfolios > MAX_PORTFOLIOS as u32
   || pallets > MAX_PALLETS as u32
   || extrinsics > MAX_EXTRINSICS as u32
{
    return Weight::MAX;
}
```

File: pallets/weights/src/pallet_capital_distribution.rs [hunk 3]
```diff
@@ -54,117 +54,168 @@ use polymesh_runtime_common::{RocksDbWeight as DbWeight, Weight};
...
-    fn claim(t: u32, w: u32) -> Weight {
-        // Minimum execution time: 384_750 nanoseconds.
-        Weight::from_ref_time(381_525_284)
-            // Standard Error: 1_803
-            .saturating_add(Weight::from_ref_time(52_168).saturating_mul(t.into()))
-            // Standard Error: 1_803
-            .saturating_add(Weight::from_ref_time(17_224).saturating_mul(w.into()))
-            .saturating_add(DbWeight::get().reads(26))
-            .saturating_add(DbWeight::get().writes(11))
+    fn claim(t: u32, _w: u32) -> Weight {
```
Comment (Neopallium) [inline pallets/weights/src/pallet_capital_distribution.rs]: New skipped param.

File: pallets/weights/src/pallet_identity.rs (hunk 2 above)
Comment (Neopallium) [reply pallets/weights/src/pallet_identity.rs]: The tx fee still needs to be higher for more complex permissions.

**Review by Neopallium — APPROVED** [review]

**Review by adamdossa — APPROVED** [review]

**Pull request merged**